### PR TITLE
Allows admin to send Candidates a message

### LINKED ISSUE #958: enhancements to sharing feedback with candidates
**Is your feature request related to a problem? Please describe.**
A need to provide more interactions with candidates of the Analytics Platform.

**Describe the solution you'd like**
Just improve the content of the message we send them and also the options for those interactions.

**Describe alternatives you've considered**
- The user still needs more interactions. Example, they might want to have a meeting with the candidate so they should be able to send a meeting request?
- Also, we need to work on the email sent to the user--- to make it more professional?
- Could we also make the email subject more positive in nature? More like: Thank you for your interest in Air Quality data?

**Additional context**
All came from user feedback.
Screenshot 2022-11-24 at 12 14 39
Screenshot 2022-11-24 at 13 22 40

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)

- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [ ] I've updated corresponding documentation for the changes in this PR.
- [ ] I have written unit and/or e2e tests for my change(s).

#### How should this be manually tested?

- git pull origin staging
- git checkout ft-candidate-chat
- npm run stage
- ON the candidates table, click the send message button and fill in the necessary message details and hit send.(Try on your personal request to be able to receive message)

#### What are the relevant tickets?

- Closes #958

#### Screenshots

message button
message email
message

### CODE REVIEW

File: netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js [hunk 1]
```diff
@@ -345,7 +400,7 @@ const CandidatesTable = (props) => {
           sendUserFeedBack(
             currentCandidate._id,
             currentCandidate.email,
-            'Your data access request has been denied',
+            'Thank you for your interest in our Air Quality data',
```
Comment (Codebmk) [inline netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js:403]: Updated rejection mail title to a more positive one:)

Comment (Codebmk) [reply netmanager/src/views/pages/CandidateList/components/CandidatesTable/CandidatesTable.js:403]: test email

**Review by 123MwanjeMike — APPROVED** [review]
Great work @Codebmk 
LGTM

**Review by Baalmart — COMMENTED** [review]
Thanks @Codebmk , this is great!

Just sharing a few feedback items. Is this PR supposed to disable the reject emails? Please see attached screenshots.

Screenshot 2022-11-25 at 13 47 07
Screenshot 2022-11-25 at 13 48 04

**Comment by Codebmk** [comment]
@Baalmart the deploy preview you used doesn't contain the most recent changes. It is still deploying

**Comment by Codebmk** [comment]
> @Baalmart the deploy preview you used doesn't contain the most recent changes. It is still deploying

Deploy preview is ready @Baalmart

**Review by Baalmart — APPROVED** [review]
Thanks @Codebmk , this looks great now1 Great work!

**Pull request merged**